apps/wallet/src/background/accounts/MnemonicAccount.ts

```diff
@@ -62,7 +62,7 @@ export class MnemonicAccount
 			sourceID,
 			address: keyPair.getPublicKey().toSuiAddress(),
 			derivationPath,
-			publicKey: keyPair.getPublicKey().toBase64(),
+			publicKey: keyPair.getPublicKey().toSuiPublicKey(),
```

Comment:
Not sure if we are releasing this anymore, and if we are if we should even change this.  I am happy to revert the wallet parts of this PR